### DISCUSSION TITLE
Viewing topic partition lag

### DISCUSSION THREAD

**Discussion by alperentvs** [provectus/kafka-ui#3429] [Q&A] [answered]
Hello,
In the current version I can view the lag of consumer groups, but not the lag of topics as a partition rather than the total lag. Is this feature available?

**Comment by Haarolean** [comment, marked as answer]
Hi, I believe it's available under certain topics within a consumer group:
![image](https://user-images.githubusercontent.com/1494347/222358930-fff1563d-874b-4823-a569-a287947604bc.png)

**Reply by alperentvs** [reply]
@Haarolean, thank you for your feedback. Finally, do these delays refresh themselves at a certain automatic time without refreshing the page, or do we have to reload the page?

**Reply by Haarolean** [reply]
you'll have to refresh the page, yeah. Btw feel free to raise an issue, we might implement it
